cluster/gce/gci/configure-helper.sh

```diff
@@ -409,6 +409,23 @@ function assemble-docker-flags {
   fi
 
   echo "DOCKER_OPTS=\"${docker_opts} ${EXTRA_DOCKER_OPTS:-}\"" > /etc/default/docker
+
+  # If using a network plugin, extend the docker configuration to always remove
+  # the network checkpoint to avoid corrupt checkpoints.
+  # (https://github.com/docker/docker/issues/18283).
+  if [[ "${use_net_plugin}" == "true" ]]; then
```

Comment:
From what I understand, I think the problem of "corrupt network checkpoint" can happen with docker networking too, and not just when the network plugins are used. But I leave that to your judgement.